packages/images/src/containers/list.js

```diff
@@ -93,14 +93,11 @@ export const List = ({
 export default compose(
   graphql(RemoveImage, { name: 'removeImage' }),
   graphql(ListImages, {
-    options: { pollInterval: 5000 },
```

Comment:
Why remove this? We need this because when we delete and go back it's still there